README.md

```diff
@@ -92,10 +94,10 @@ docker run \
     --env OTEAPI_REDIS_TYPE=redis \
     --env OTEAPI_REDIS_HOST=redis \
     --env OTEAPI_REDIS_PORT=6379 \
-    ontotrans/oteapi:latest
+    emmc-asbl/oteapi:latest
 ```
 
-Open the following URL in a browser [http://localhost:80/redoc](http://localhost:80/redoc).
+Open the following URL in a browser [http://localhost/redoc](http://localhost:80/redoc).
```

Comment:
According to Dockerfile and entrypoint.sh runs production also on port 8080
```suggestion
Open the following URL in a browser [http://localhost:8080/redoc](http://localhost:8080/redoc).
```

Reply:
Are you sure? I think for this particular example it should map to port `80`?

Reply:
For production the README maps the `8080` port within the docker container to port `80` on the host.